plugins/ruby-foreman-templates/debian/control

```diff
@@ -8,7 +8,7 @@ Homepage: https://github.com/theforeman/foreman_templates
 
 Package: ruby-foreman-templates
 Architecture: all
-Depends: ${misc:Depends}, bundler, foreman (>= 1.12.0), git
+Depends: ${misc:Depends}, bundler, foreman (>= 1.14.1), git
```

Comment:
No, there was a minor fix to exporting capabilities on the Foreman core side that we make use of.